bluesky/run_engine.py

```diff
@@ -1258,9 +1258,11 @@ def _monitor(self, msg):
                                              obj))
         descriptor_uid = new_uid()
         data_keys = obj.describe()
-        config = obj.read_configuration()
-        for key, val in list(config.items()):
-            val['value'] = sanitize_np(val['value'])
+        config = {obj.name: {'data': {}, 'timestamps': {}}}
+        config[obj.name]['data_keys'] = obj.describe_configuration()
+        for key, val in list(obj.read_configuration().items()):
+            config[obj.name]['data'][key] = sanitize_np(val['value'])
+            config[obj.name]['data'][key] = val['timestamp']
```

Comment:
No, the structure of `read()` and `read_configuration()` is `{<data_key>: {'value': ..., 'timestamp': ...}, ...}`